docs/UserGuide.md

```diff
@@ -93,7 +93,25 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd food item in the food inventory.
 * `find k/tuna` followed by `delete 1` deletes the 1st food item in the results of the `find` command.
 
-### Clearing all entries : `clear` [coming soon]
+### Editing a food item : `edit`
+
+Edits the details of an existing food item in the food inventory.
+
+Format: `edit INDEX [d/DESCRIPTION] [p/PRIORITY] [e/EXPIRY DATE] [t/TAG]...`
+
+* Edits the food item at the specified `INDEX`. 
+* The index refers to the index number shown in the displayed food item list. 
+* The index **must be a positive integer** 1, 2, 3, …​
+* At least one of the fields must be provided.
```

Comment:
Very minor, but what do you think of "At least one of the optional fields must be provided."? Since INDEX is not optional.

Reply:
Noted, thanks!